playground/terraform/infrastructure/memorystore/variables.tf

```diff
@@ -48,7 +48,7 @@ variable "tier" {
 
 variable "replicas_mode" {
   description = "If enabled read endpoint will be provided and the instance can scale up and down the number of replicas"
-  default     = "READ_REPLICAS_ENABLED"
+  default     = "READ_REPLICAS_DISABLED"
```

Comment:
Why did we change our default here?

Reply:
Since we use Redis to "BASIC" mode, Read_Replica is not available for it